ansible_builder/main.py

```diff
@@ -32,12 +32,13 @@ def __init__(self, action=None,
                  verbosity=0):
         self.action = action
         self.definition = UserDefinition(filename=filename)
+        self.base_image = self.definition.raw.get('base_image') or base_image
```

Comment:
I don't know about the precedence. If I provide the base image both in the definition file, and with the CLI option... I would think the CLI option should win. That tends to be how we want config processing to go. @shanemcd what do you think? Or else we could error if given both.

We would have to re-think how the kwarg default works, it might make more sense for `None` to be the value from the kwarg.

Reply:
> If I provide the base image both in the definition file, and with the CLI option... I would think the CLI option should win.

Agreed, I'll switch those around and clarify the precedence in the docs.

Reply:
Have made some updates, will work on making sure integration tests pass next.